### PR TITLE
Add duct tape blindfold for old reciepe plus tweaks for reciepe

### PR DESCRIPTION
**Pull request by Lamandus**
#### Summary
Balance "Adding new duct tape blindfold and changes old blindfold (out of duct tape) reciepe to make more sense"

#### Purpose of change
In the old reciepe, you could make a blindfold out of duct tape. Its component has a volume of around 0.01 ml and made an item with 0,25 ml. Also it changed from plastic to cotton, which also doesn't make any sense.

#### Describe the solution
I made a own item based on the old blindfold but with tweaked values. The new duct tape blindfold is smaller and lighter. Also you don't need any know how to produce it (when you were able to make makeshift knifes out of no knowledge, you can do that too). Also the new "old" reciepe uses 50 duct tape, instead of only 10.

#### Testing
works as intended.

### CODE REVIEW

File: data/json/recipes/armor/head.json [hunk 1]
```diff
@@ -53,7 +53,7 @@
     "time": "3 m",
     "reversible": true,
```
Comment (actual-nh) [inline data/json/recipes/armor/head.json:54]: Hrm. Should it be entirely reversible?

Comment (Lamandus) [reply data/json/recipes/armor/head.json:54]: good catch, I delete the line.

File: data/json/recipes/armor/head.json [hunk 2]
```diff
@@ -42,18 +42,17 @@
...
-    "result": "blindfold",
+    "result": "blindfold_duct",
     "type": "recipe",
     "activity_level": "NO_EXERCISE",
     "id_suffix": "from_tape",
     "category": "CC_ARMOR",
     "subcategory": "CSC_ARMOR_HEAD",
     "skill_used": "fabrication",
-    "difficulty": 1,
+    "difficulty": 0,
```
Comment (ZhilkinSerg) [inline data/json/recipes/armor/head.json:52]:
```suggestion
```

File: data/json/recipes/armor/head.json [hunk 3]
```diff
@@ -42,18 +42,16 @@
     "components": [ [ [ "felt_patch", 4 ] ] ]
   },
   {
-    "result": "blindfold",
+    "result": "blindfold_duct",
```
Comment (ZhilkinSerg) [inline data/json/recipes/armor/head.json:45]:
```suggestion
    "result": "blindfold",
```

**Pull request merged**